sigle/src/modules/editor/components/SlateEditor.tsx

```diff
@@ -594,52 +592,68 @@ export const SlateEditor = ({
-              <ButtonOutline style={{ marginRight: 6 }} disabled>
-                Saving ...
-              </ButtonOutline>
-            )}
-            {!loadingSave && story.type === 'public' && (
-              <ButtonOutline
-                style={{ marginRight: 6 }}
-                onClick={() => handleSave()}
-              >
+    <Container>
+      <Flex
+        as="header"
+        justify="between"
+        align="center"
+        css={{
+          mt: '$5',
+
+          '@md': {
+            mt: '$10',
+          },
+        }}
+      >
+        <Flex gap="10" align="center">
+          <Link href="/" passHref>
+            <a>
+              <ArrowLeftIcon />
+            </a>
+          </Link>
+          <Text size="sm">
+            <Box as="span" css={{ fontWeight: 'bold' }}>{user?.username}</Box>
+            <span>{` | Draft`}</span>
+          </Text>
+        </Flex>
+        <Flex gap="10">
+          {loadingSave && (
+            <Button disabled variant="ghost">
+              Saving ...
+            </Button>
+          )}
+          {!loadingSave && story.type === 'public' && (
+            <Button variant="ghost">Save</Button>
+          )}
+          {!loadingSave && story.type === 'private' && (
+            <Tippy
+              content="Nobody can see it unless you click on « publish »"
+              theme="light-border"
+            >
+              <Button onClick={() => handleSave()} variant="ghost">
                 Save
-              </ButtonOutline>
-            )}
-            {!loadingSave && story.type === 'private' && (
-              <Tippy
-                content="Nobody can see it unless you click on « publish »"
-                theme="light-border"
-              >
-                <ButtonOutline
-                  style={{ marginRight: 12 }}
-                  onClick={() => handleSave()}
-                >
-                  Save
-                </ButtonOutline>
-              </Tippy>
-            )}
-            {story.type === 'private' && (
-              <ButtonOutline style={{ marginRight: 6 }} onClick={onPublish}>
-                Publish
-              </ButtonOutline>
-            )}
-            {story.type === 'public' && (
-              <ButtonOutline style={{ marginRight: 6 }} onClick={onUnpublish}>
-                Unpublish
-              </ButtonOutline>
-            )}
-            <AppBarSettings onClick={handleOpenSettings}>
-              <MdSettings size={22} />
-            </AppBarSettings>
-          </StyledAppBarRightContainer>
-        </AppBar>
-      </FixedContainer>
+              </Button>
+            </Tippy>
+          )}
+          {story.type === 'private' && (
+            <Button onClick={onPublish} variant="ghost">
+              Publish
+            </Button>
+          )}
+          {story.type === 'public' && (
+            <Button onClick={onUnpublish} variant="ghost">
+              Publish
+            </Button>
+          )}
+          <Button
```

Comment:
I missed this one, but this should be using `IconButton`, that way it will have the proper padding :)